flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java

```diff
@@ -438,7 +617,7 @@ public void close() throws Exception {
 	}
 
 	@SuppressWarnings("unchecked")
-	private static class ResultSortComparator implements Comparator<Object> {
+	private static class Tuple2ResultSortComparator implements Comparator<Object> {
```

Comment:
this is also a duplicate with `NonKeyedWindowOperatorTest` according to IntelliJ